specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProfile.json

```diff
@@ -69,327 +82,540 @@
             "Disabled",
-            "Warned"
+            "Warned",
+            "Deleted",
+            "UnderReview"
           ],
-          "readOnly": true,
           "type": "string",
+          "readOnly": true,
           "x-ms-enum": {
-            "modelAsString": true,
-            "name": "BillingProfileStatus"
+            "name": "BillingProfileStatus",
+            "modelAsString": true
           }
         },
         "statusReasonCode": {
           "description": "Reason for the specified billing profile status.",
           "enum": [
             "PastDue",
+            "UnusualActivity",
             "SpendingLimitReached",
             "SpendingLimitExpired"
           ],
-          "readOnly": true,
           "type": "string",
+          "readOnly": true,
           "x-ms-enum": {
-            "modelAsString": true,
-            "name": "StatusReasonCode"
+            "name": "BillingProfileStatusReasonCode",
+            "modelAsString": true
           }
         },
         "systemId": {
           "description": "The system generated unique identifier for a billing profile.",
-          "readOnly": true,
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "tags": {
+          "description": "Tags of billing profiles.",
+          "type": "object",
           "additionalProperties": {
             "type": "string"
-          },
-          "description": "Tags of billing profiles.",
-          "maxItems": 1000,
-          "type": "object"
+          }
         },
         "targetClouds": {
           "description": "Identifies the cloud environments that are associated with a billing profile. This is a system managed optional field and gets updated as the billing profile gets associated with accounts in various clouds.",
+          "type": "array",
           "items": {
-            "$ref": "#/definitions/TargetCloud"
+            "description": "Identifies the cloud environments that are associated with an invoice section. This is a system managed optional field and gets updated as the invoice section gets associated with accounts in various clouds.",
+            "enum": [
+              "Unknown",
+              "AirgapTest",
+              "USGov",
+              "USNat",
+              "USSec"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "TargetCloud",
+              "modelAsString": true
+            }
           },
           "readOnly": true,
-          "type": "array"
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "DeleteBillingProfileEligibilityDetail": {
+      "description": "Validation details of delete billing profile eligibility.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Code of the delete invoice section eligibility response.",
+          "enum": [
+            "None",
+            "AzureSubscriptions",
```

Comment:
Let's avoid using the term **Azure** subscriptions if we can... Can we either use "Subscriptions" or "Billing Subscriptions"?

Reply:
Also, why do we need a separate value for this compared to ActiveBillingSubscriptions anyway?

Reply:
From what I can tell `AzureSubscriptions` is a subset of `ActiveBillingSubscriptions`... 
@ramkd12 @amberbhargava to comment on guidance here?

Reply:
Remove in favor of ActiveBillingSubscriptions